src/audio/base_fw.c

```diff
@@ -71,7 +71,7 @@ static int basefw_config(uint32_t *data_offset, char *data)
 
 	/* TODO: add log support */
 	tuple = next_tuple(tuple);
-	set_tuple_uint32(tuple, IPC4_TRACE_LOG_BYTES_FW_CFG, 0);
+	set_tuple_uint32(tuple, IPC4_TRACE_LOG_BYTES_FW_CFG, 1024);
```

Comment:
Why temporary, is it pending on kernel PRs merging ? Fwiw, -1 is probably a better value here as it will never be a realistic runtime position (unless mailbox becomes 4G).

Reply:
@lgirdwood we need the full implementation on FW side before this change can be merge. If this is non-zero, kernel thinks FW supports mtrace (which is currently does not).  I just pushed another draft to Zephyr -> https://github.com/zephyrproject-rtos/zephyr/pull/47058

Reply:
@kv2019i, it depends on what you read, but the IPC4_TRACE_LOG_BYTES_FW_CFG is defined as
A) the **sum** of the sizes of all slots in the debug window
B) size of a single slot

And this is kind of random and likely platform dependent...
In any case, I would set it to `0x1000` as this is the size of a single slot and this is always true.